src/coreComponents/physicsSolvers/fluidFlow/CompositionalMultiphaseBase.cpp

```diff
@@ -365,49 +365,17 @@ void CompositionalMultiphaseBase::registerDataOnMesh( Group & meshBodies )
     {
       if( m_hasCapPressure )
       {
-        subRegion.registerWrapper< string >( viewKeyStruct::capPressureNamesString() ).
-          setPlotLevel( PlotLevel::NOPLOT ).
-          setRestartFlags( RestartFlags::NO_WRITE ).
-          setSizedFromParent( 0 ).
-          setDescription( "Name of the capillary pressure constitutive model to use" ).
```

Comment:
We have lost these descriptions. But I suppose they aren't user accessible.

Reply:
Isn't the description also used in the docs describing the c++ components though?  Maybe not for these because of where it's happening.